api/main.go

```diff
@@ -132,19 +137,42 @@ func main() {
 
 	setupRestful(*lgc)
 
-	// Since this is ECS, we run a right sizer for the clusters to kill off
-	// unused clusters
-	logrus.Infof("Starting RightSizer")
-	backend.StartRightSizer()
-
-	deployLogic := logic.NewL0DeployLogic(*lgc)
 	taskLogic := logic.NewL0TaskLogic(*lgc)
+	deployLogic := logic.NewL0DeployLogic(*lgc)
 	jobLogic := logic.NewL0JobLogic(*lgc, taskLogic, deployLogic)
+	environmentLogic := logic.NewL0EnvironmentLogic(*lgc)
+
 	jobJanitor := logic.NewJobJanitor(jobLogic)
+	go runEnvironmentScaler(environmentLogic)
 
 	logrus.Infof("Starting Job Janitor")
 	jobJanitor.Run()
 
 	logrus.Print("Service on localhost" + port)
 	logrus.Fatal(http.ListenAndServe(port, nil))
 }
+
+func runEnvironmentScaler(environmentLogic *logic.L0EnvironmentLogic) {
```

Comment:
essentially the new Right Sizer, sleeps for 5 mins in a separate goroutine and then scales things accordingly